testing/asserts.ts

```diff
@@ -469,15 +472,15 @@ export function assertNotMatch(
  * If not, then throw.
  */
 export function assertObjectMatch(
-  actual: Record<PropertyKey, unknown>,
+  actual: Record<PropertyKey, any>,
```

Comment:
I think you need to add `// deno-lint-ignore no-explicit-any` to pass the linter check.
Should the `expected` variable keep the typing `Record<PropertyKey, unknown>` or should it also use `any` instead of `unknown`?

Reply:
This is due to an intentional quirk in how `TypeScript` operates with `interfaces` 

- microsoft/TypeScript#42825
- microsoft/TypeScript#41518

Reply:
Actually I was wondering why just `actual` is being changed while `expected` is left as is.
Shouldn't both be modified?

Reply:
The problem occurs for the first `params` `actual` because it would be `interface` or `types` there is no need to change the second param to `any`.